web/src/components/PendingProjects/PendingProjects.tsx

```diff
@@ -32,15 +32,19 @@ function PendingProjects({
   useEffect(() => {
     getAllApps().then(
       (results: {
-        [app_id: string]: InstalledAppInfo & { cellIdString: string }
+        [app_id: string]: AppInfo & { cellIdString: string }
       }) => {
         const newPassphrases: AppInfoStore = {}
         pendingProjects.forEach((pendingCellId) => {
           const [appId, appInfo] = Object.entries(results).find(
             ([_appId, appInfo]) => appInfo.cellIdString === pendingCellId
           )
+
```

Comment:
This bit of code isn't valid. Need to revise it like we did elsewhere. Test it by using 'Join a project', and inputting any secret phrase that doesn't exist